src/presentation/dtos/media-dto.ts

```diff
@@ -0,0 +1,9 @@
+import {Media} from '../../model/entities/media'
+
+
+export class MediaDto {
```

Comment:
Just a heads up. This has also been introduced in https://github.com/tv2/sofie-server/pull/83
So we need to be aware that we only end up with one MediaDto

Reply:
I'll look out for it!